### DISCUSSION TITLE
Cinemata - https://cinemata.org/

### DISCUSSION THREAD

**Discussion by mgogoulos** [mediacms-io/mediacms#818] [Show and tell]
![Screenshot from 2023-06-26 16-09-49](https://github.com/mediacms-io/mediacms/assets/1027116/a0b67112-f8a2-4f76-b5bc-c86831d0e69b)

**Comment by mgogoulos** [comment]
https://cinemata.org/ is part of Engage Media,  a non-profit media, technology, and culture organisation. The portal has been up since 2021, hosts a few thousand videos mostly uploaded by producers and is curated by a whole team. Cinemata portal is having extra features comparing to vanilla MediaCMS, some of them are:

* a featured video in the front page
* a banner showing information messages
* a poster pop up that displays an image with text, set by the editorial team
* first page includes more video/media category, not just latest/featured/recomended
* srt subtitles are allowed, and are converted to webvtt
* a simple subtitle editor
* videos are optionally getting an automated transcript and translation, powered by OpenAI's Whisper
* Top level categories, as Languages, Countries, Topics

**Reply by egiaggata** [reply]
hi, I am studying for mediacms, for comparison can i use source code of cinemata.org?.

Thanks

**Reply by mgogoulos** [reply]
The source code for cinemata.org is not available

**Comment by kayzworld** [comment]
Hello Mgogoulos, how can I customize my cms like that of cinemata.org? Also, I have tried basic customization like background-color change on the _common.css file and no effect is taking place. What am I missing out?
